pyproject.toml

```diff
@@ -5,16 +5,23 @@ description = "Dagster ETL for Wikipedia data."
 authors = ["imlapps"]
 readme = "README.md"
 
+[tool.mypy]
+python_version = "3.10"
```

Comment:
comment: I've been moving my stuff to 3.12.

Also, I use separate `mypy.ini` and `ruff.toml` files in new projects so I can copy them as-is to other projects, instead of slotting them into `pyproject.toml`.

Reply:
That makes sense. I have separated the files for this project, and will do same in future projects.